tests/lib/prepare-restore.sh

```diff
@@ -513,6 +513,7 @@ prepare_suite_each() {
     # shellcheck source=tests/lib/reset.sh
     "$TESTSLIB"/reset.sh --reuse-core
     # Reset systemd journal cursor.
+    sync_journalctl_log
```

Comment:
Would it be more sensible to move that functionality into `start_new_journalctl_log`?  This way there are fewer functions to keep track of.

Reply:
Thanks for the review!!! 
change done